### DISCUSSION TITLE
241121_목

### DISCUSSION THREAD

**Discussion by wonjeongjeong** [boostcampaitech7/level2-cv-semanticsegmentation-cv-19-lv3#41] [TIL]
- [x] smp 실험 augmentation 추가해서 해보기
- [ ] 백본 좀 더 찾아보기

**Comment by june21a** [comment]
- [x] mmsegmentation segformer하고 segnext augmentation 실험
- [ ] YOLO 시도해보기

**Comment by Yoon0717** [comment]
- [x] Loss 실험
- [x] Scheduler 실험
- [ ] Augmentation 실험

**Comment by Soy17** [comment]
- [ ] Pseudo labeling 실험 진행
- [ ] 과제 마무리

**Comment by sejongmin** [comment]
- [x] sam2unet size 실험
- [x] augmentation 실험
- [x] wandb 추가하기
